Type Collision when using SkyApm.Diagnostics.MongoDB and MongoDB.Driver

Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [x] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Bug

Using Dotnet 8 and SkyApm.Diagnostics.MongoDB Version 2.2.0.

When using MongoDB.Driver nuget with SkyApm.Diagnostics.MongoDB, there is a type collision for certain types. Steps to reproduce:

```bash
dotnet new console -o MongoTypeCollisions
cd MongoTypeCollisions
dotnet add package MongoDB.Driver --version 3.0.0
dotnet add package SkyApm.Diagnostics.MongoDB --version 2.2.0
echo "using MongoDB.Driver; CollectionNamespace collectionNamespace= new();" > Program.cs
dotnet build
```
I get the following error message:
```
/home/emilioz/personal/MongoTypeCollisions/Program.cs(1,23): error CS0433: The type 'CollectionNamespace' exists in both 'MongoDB.Driver.Core, Version=2.18.0.0, Culture=neutral, PublicKeyToken=null' and 'MongoDB
.Driver, Version=3.0.0.0, Culture=neutral, PublicKeyToken=94992a530f44e321' [/home/emilioz/personal/MongoTypeCollisions/MongoTypeCollisions.csproj]
```

I haven't looked at this codebase, but I suspect updating to MongoDB.Driver.Core 2.30.0 (the latest version) might fix the issue.

--- a/src/SkyApm.Diagnostics.MongoDB/SkyApm.Diagnostics.MongoDB.csproj
+++ b/src/SkyApm.Diagnostics.MongoDB/SkyApm.Diagnostics.MongoDB.csproj
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <ItemGroup>
-        <PackageReference Include="MongoDB.Driver.Core" Version="2.18.0" />
+        <PackageReference Include="MongoDB.Driver.Core" Version="2.30.0" />
     </ItemGroup>
   
     <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">

